faqs-january-28-2017.html

```diff
@@ -0,0 +1,96 @@
+---
+layout: page
+title: YourHaq - Frequently Asked Questions (FAQs) on President Trump’s Refugee Executive Order 
+permalink: /faqs-executive-order/
+---
+<!-- Slider Start -->
+<section id="global-header">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="block">
+          <h1 class="animated fadeInUp">Frequently Asked Questions on President Trump’s Refugee Executive Order</h1>
+          <p>NOTE: This document is not legal advice but instead provides the best answers that we can provide at this time. It is subject to change as additional guidance is released from the U.S. Government. Additionally, the legality of the actions in the President’s Executive Order are subject to judicial review. Please be aware that certain aspects of the order may be contested for its constitutionality.</p>
```

Comment:
Would be nice to remove/truncate this disclaimer, in favor of the one already present in `footer.html`. Reading this line does raise the question of whether disclaimer in `footer.html` should be updated? /cc @aymannadeem